Zend/tests/type_declarations/mixed/inheritance/mixed_return_inheritance_success4.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Test that a no return type can be overridden by the mixed type
```

Comment:
Hm, nowadays: `no return`, is it `none`, `mixed` or `void`?

Reply:
No return is `mixed|void`accoring to this RFC :) And I like the `mixed` name, especially because it's what's used all over the documentation and static analyzers.